plasmapy/particles/particle_class.py

```diff
@@ -413,10 +413,19 @@ class Particle(AbstractPhysicalParticle):
     def __init__(
         self,
         argument: ParticleLike,
+        *_,
         mass_numb: Integral = None,
         Z: Integral = None,
     ):
 
+        # TODO: Remove the following block during or after the 0.9.0 release
+
+        if _:  # coverage: ignore
```

Comment:
I don't think this should be ignored for tests.  Since we're "mimicking" Python's default behavior for `foo(x, *, bar=None)`, we should test that it is behaving that way.  ... I can't really solidify my arguments for this, but my gut is telling me it should be tested.

Reply:
Sure thing!  My conscience was telling me I probably ought to have tested that block anyway...

 - [ ] Do this ↑